### PR TITLE
lint refresh

### PR DESCRIPTION
**Pull request by ssbarnea**
Needed for avoiding a false positive of:
```
[BLK100] Black would make changes.
```
This was happening because flake8-black adopted new format of black but we were using an older version of black. They need to be kept in sync in order to avoid such errors. Arguably we could even remove flake8-black in the future because we always run black anyway.

### CODE REVIEW

**Comment by gundalow** [comment]
closed/reopen to try and tick Travis-CI back into life

File: molecule/test/unit/test_config.py [hunk 1]
```diff
@@ -376,7 +376,7 @@ def test_set_env_from_file(config_instance):
 
 
 def test_set_env_from_file_returns_original_env_when_env_file_not_found(
-    config_instance
+    config_instance,
```
Comment (sshnaidm) [inline molecule/test/unit/test_config.py]: And how is that related?

Comment (ssbarnea) [reply molecule/test/unit/test_config.py]: Black formatting changed, that was made by black, not me.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -17,11 +17,11 @@ repos:
       - id: debug-statements
         language_version: python3
   - repo: https://gitlab.com/pycqa/flake8.git
-    rev: 3.7.8
+    rev: 3.7.9
     hooks:
       - id: flake8
         additional_dependencies:
-          - flake8-black
```
Comment (ssbarnea) [inline .pre-commit-config.yaml]: This was tricky because a cached version may have produced different results.
